tests/libres_tests/res/enkf/test_field_export.py

```diff
@@ -46,7 +46,7 @@ def test_field_basics(self):
             self.assertEqual(grid.get_num_active(), len(field_node))
             with self.assertRaises(IndexError):
                 field_node[grid.get_num_active()]
-            value0 = field_node[0]
+            field_node[0]
```

Comment:
Maybe delete.

Reply:
I think this dives into cwrap, so I think I will keep it.